gamewindow.qrc

```diff
@@ -72,8 +72,8 @@
         <file>resources/sound/GameStart.wav</file>
         <file>resources/sound/Mill_B.wav</file>
         <file>resources/sound/Mill_W.wav</file>
-        <file>resources/sound/MillRepeatly_B.wav</file>
-        <file>resources/sound/MillRepeatly_W.wav</file>
+        <file>resources/sound/MillRepeatedly_B.wav</file>
+        <file>resources/sound/MillRepeatedly_W.wav</file>
```

Comment:
This was confusing. But the code expects this spelling, so it's very much the right thing to do....